src/graph_config.js

```
@@ -570,34 +570,56 @@ GraphConfig.getDefaultCurveForField = function (flightLog, fieldName) {
-              };
-            case "debug[3]": // vario
-              return {
-                power: 1,
-                MinMax: {
-                  min: -5,
-                  max: 5,
-                },
-              };
-            default:
-              return getCurveForMinMaxFields(fieldName);
+          if (semver.gte(flightLog.getSysConfig().firmwareVersion, "2026.6.0")) {
```

ESLint

[error] 'semver' is not defined.

(no-undef)